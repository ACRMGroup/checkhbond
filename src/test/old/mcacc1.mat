residue ARG
accept	      34	      30	      30	    17
partnertoaccept	      39	      28	      28	     1
residue THR
accept	      34	      30	      30	    14
partnertoaccept	      38	      30	      28	     1
residue ASN
accept	      34	      30	      30	     9
residue ASP
accept	      34	      30	      30	     8
partnertoaccept	      35	      25	      30	     1
residue GLU
accept	      34	      30	      30	    11
partnertoaccept	      37	      26	      32	     1
residue GLN
accept	      34	      30	      30	     7
partnertoaccept	      39	      27	      32	     1
residue LYS
accept	      34	      30	      30	     5
partnertoaccept	      33	      26	      30	     1
partnertoaccept	      35	      27	      26	     1
partnertoaccept	      36	      30	      24	     1
residue SER
accept	      34	      30	      30	    17
residue TRP
accept	      34	      30	      30	     1
residue TYR
accept	      34	      30	      30	     8
partnertoaccept	      33	      26	      34	     1
residue HIS
accept	      34	      30	      30	     7
residue HIS
accept	      34	      30	      30	     7
residue ALA
accept	      34	      30	      30	     7
partnertoaccept	      36	      26	      30	     1
residue CYS
accept	      34	      30	      30	    10
partnertoaccept	      38	      30	      34	     1
residue PHE
accept	      34	      30	      30	     5
residue GLY
accept	      34	      30	      30	    13
partnertoaccept	      37	      28	      34	     1
partnertoaccept	      38	      26	      30	     1
partnertoaccept	      38	      27	      29	     1
partnertoaccept	      40	      29	      28	     1
residue ILE
accept	      34	      30	      30	     6
residue LEU
accept	      34	      30	      30	    14
residue MET
accept	      34	      30	      30	     6
partnertoaccept	      35	      24	      28	     1
partnertoaccept	      35	      27	      25	     1
partnertoaccept	      37	      29	      35	     1
residue VAL
accept	      34	      30	      30	    15
partnertoaccept	      33	      24	      28	     1
residue PRO
accept	      34	      30	      30	    14
partnertoaccept	      35	      25	      33	     1
partnertoaccept	      36	      27	      34	     1
