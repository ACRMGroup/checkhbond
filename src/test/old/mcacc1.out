INFO: Processing residue type ARG
INFO: Processing file /tmp//1tsr
THR    284 OG1  : ARG    280 O     ??? C   
INFO: Processing residue type THR
INFO: Processing file /tmp//1tsr
ARG    283 NH2  : THR    118 O     ??? C   
INFO: Processing residue type ASN
INFO: Processing file /tmp//1tsr
INFO: Processing residue type ASP
INFO: Processing file /tmp//1tsr
SER    183 OG   : ASP    184 O     ??? C   
INFO: Processing residue type GLU
INFO: Processing file /tmp//1tsr
THR    230 OG1  : GLU    221 O     ??? C   
INFO: Processing residue type GLN
INFO: Processing file /tmp//1tsr
THR    170 OG1  : GLN    167 O     ??? C   
INFO: Processing residue type LYS
INFO: Processing file /tmp//1tsr
ARG    267 NE   : LYS    101 O    HE   C   
ARG    267 NH2  : LYS    101 O     ??? C   
ASN    235 ND2  : LYS    139 O     ??? C   
INFO: Processing residue type SER
INFO: Processing file /tmp//1tsr
INFO: Processing residue type TRP
INFO: Processing file /tmp//1tsr
INFO: Processing residue type TYR
INFO: Processing file /tmp//1tsr
SER    149 OG   : TYR    107 O     ??? C   
INFO: Processing residue type HIS
INFO: Processing file /tmp//1tsr
INFO: Processing residue type HIS
INFO: Processing file /tmp//1tsr
INFO: Processing residue type ALA
INFO: Processing file /tmp//1tsr
SER    185 OG   : ALA    189 O     ??? C   
INFO: Processing residue type CYS
INFO: Processing file /tmp//1tsr
SER    116 OG   : CYS    124 O     ??? C   
INFO: Processing residue type PHE
INFO: Processing file /tmp//1tsr
INFO: Processing residue type GLY
INFO: Processing file /tmp//1tsr
ARG    110 NE   : GLY    108 O    HE   C   
ARG    110 NH2  : GLY    108 O     ??? C   
THR    125 OG1  : GLY    117 O     ??? C   
ARG    249 NH2  : GLY    245 O     ??? C   
INFO: Processing residue type ILE
INFO: Processing file /tmp//1tsr
INFO: Processing residue type LEU
INFO: Processing file /tmp//1tsr
INFO: Processing residue type MET
INFO: Processing file /tmp//1tsr
ARG    175 NE   : MET    237 O    HE   C   
ARG    175 NH2  : MET    237 O     ??? C   
ARG    249 NE   : MET    246 O    HE   C   
INFO: Processing residue type VAL
INFO: Processing file /tmp//1tsr
SER    240 OG   : VAL    274 O     ??? C   
INFO: Processing residue type PRO
INFO: Processing file /tmp//1tsr
THR    155 OG1  : PRO    152 O     ??? C   
ARG    175 NH1  : PRO    191 O     ??? C   
